src/mariadb.ts

```diff
@@ -0,0 +1,68 @@
+const mariadb = require("mariadb");
+const uuidv4 = require('uuid/v4');
+
+export class MariaDB {
+    constructor() {
+
+    }
+
+    public async register(username: string, password: string) {
+        const conn = await this.createConnection()
+        const otherUsers = await conn.query(
+            `SELECT * 
+             FROM Users
+             WHERE UserName="${username}"`
+        );
+        if (otherUsers.length > 0) {
+            conn.end();
+            throw new Error('Username is taken');
+        }
+        await conn.query(
+		    `INSERT INTO Users (UserName, Password, Credit) values (
+			    "${username}",
+                "${password}",
+                0
+            )`);
```

Comment:
there just seems to be some weird formatting here, but it could just be in github